pkg/api/api_test.go

```diff
@@ -5,10 +5,11 @@ import (
 	"net/http/httptest"
 	"net/url"
 
+	"github.com/gorilla/mux"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
-	profilesv1 "github.com/weaveworks/profiles/api/v1alpha1"
+	"github.com/weaveworks/profiles/api/v1alpha1"
```

Comment:
bad merge? I think we want `profilesv1` :smile:

Reply:
ugh that merge was crap